examples/src/main/scala/org/apache/spark/examples/streaming/ActorWordCount.scala

```diff
@@ -63,11 +63,11 @@ class FeederActor extends Actor {
   def receive: Receive = {
     case SubscribeReceiver(receiverActor: ActorRef) =>
       println("received subscribe from %s".format(receiverActor.toString))
-      receivers = LinkedList(receiverActor) ++ receivers
+      receiverActor +=: receivers
 
     case UnsubscribeReceiver(receiverActor: ActorRef) =>
       println("received unsubscribe from %s".format(receiverActor.toString))
-      receivers = receivers.dropWhile(x => x eq receiverActor)
+      receivers.dropWhile(x => x eq receiverActor)
```

Comment:
The difference between the two:

> The iterator and all traversal methods of this class visit elements in the order they were inserted. 

So it really depends if ordering is important or not